core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionValidatorSpec.scala

```diff
@@ -61,19 +77,19 @@ class ExpressionValidatorSpec extends FlatSpec with TestDriverComponent with Exp
 
   it should "validateAndParseMCExpressions" in {
```

Comment:
Should there be some `allowRootEntity = false` test cases here?

Reply:
I guess so. I'll doublecheck the code path here, the function names don't tell me much about the difference between `validateAndParse` and `validateAndParseMCExpressions`